pkg/util/overridemanager/overridemanager.go

```diff
@@ -121,26 +127,26 @@ func (o *overrideManagerImpl) applyNamespacedOverrides(rawObj *unstructured.Unst
 		return nil, nil
 	}
 
-	matchingPolicies := o.getMatchingOverridePolicies(policyList.Items, rawObj, cluster)
-	if len(matchingPolicies) == 0 {
+	matchingPolicyOverriders := o.getMatchingOverridePolicies(policyList.Items, rawObj, cluster)
+	if len(matchingPolicyOverriders) == 0 {
 		klog.V(2).Infof("No override policy for resource(%s/%s)", rawObj.GetNamespace(), rawObj.GetName())
 		return nil, nil
 	}
 
 	appliedList := &AppliedOverrides{}
-	for _, p := range matchingPolicies {
-		if err := applyPolicyOverriders(rawObj, p.Spec.Overriders); err != nil {
-			klog.Errorf("Failed to apply overrides(%s/%s) for resource(%s/%s), error: %v", p.Namespace, p.Name, rawObj.GetNamespace(), rawObj.GetName(), err)
+	for _, p := range matchingPolicyOverriders {
+		if err := applyPolicyOverriders(rawObj, p.overriders); err != nil {
+			klog.Errorf("Failed to apply overrides(%s/%s) for resource(%s/%s), error: %v", p.namespace, p.name, rawObj.GetNamespace(), rawObj.GetName(), err)
 			return nil, err
 		}
-		klog.V(2).Infof("Applied overrides(%s/%s) for resource(%s/%s)", p.Namespace, p.Name, rawObj.GetNamespace(), rawObj.GetName())
-		appliedList.Add(p.Name, p.Spec.Overriders)
+		klog.V(2).Infof("Applied overrides(%s/%s) for resource(%s/%s)", p.namespace, p.name, rawObj.GetNamespace(), rawObj.GetName())
+		appliedList.Add(p.name, p.overriders)
 	}
 
 	return appliedList, nil
 }
 
-func (o *overrideManagerImpl) getMatchingClusterOverridePolicies(policies []policyv1alpha1.ClusterOverridePolicy, resource *unstructured.Unstructured, cluster *clusterv1alpha1.Cluster) []policyv1alpha1.ClusterOverridePolicy {
+func (o *overrideManagerImpl) getMatchingClusterOverridePolicies(policies []policyv1alpha1.ClusterOverridePolicy, resource *unstructured.Unstructured, cluster *clusterv1alpha1.Cluster) []policyOverriders {
```

Comment:
```suggestion
func (o *overrideManagerImpl) getOverridersFromClusterOverridePolicies(policies []policyv1alpha1.ClusterOverridePolicy, resource *unstructured.Unstructured, cluster *clusterv1alpha1.Cluster) []policyOverriders {
```

Should we update the method name as well?

Reply:
Yes, somehow I missed this.